Parsers/ASimNetworkSession/Parsers/ASimNetworkSession.yaml

```diff
@@ -1,6 +1,6 @@
 Parser:
   Title: Network Session ASIM parser
-  Version: '0.3'
+  Version: '0.4'
```

Comment:
You did not update the version in all of them, neither the date (including here)